### PR TITLE
CLI: Removed -l option add a --quiet option

### PR DESCRIPTION
**Pull request by radmirnovii**
Flag -l removed, this behaviour was made default
Flag -q --quiet added.

Resolved: vmware#490

Signed-off-by: Radmir Mukhambetov <radmirnovii@gmail.com>

### CODE REVIEW

File: docs/releases/v0_4_0.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Summary
 Although the focus of this release was on enabling SPDX formatted reports, a good chunk of work went into resolving technical debt. Most notably Michael Rohan's work towards cleaning up code style and enabling a workflow for contributors to check for linting errors. There was also a lot of work by Santiago Torres towards making Tern pip installable.
 
-Tern now supports the SPDX document format. As of this release the documents produced are valid SPDX. You can try it out by running `tern -l report -i <docker image> -m spdxtagvalue -f spdx-container-image.txt`. You can validate the document using [SPDX tools](https://github.com/spdx/tools). Note that python tools are still a work in progress but you can get on the [mailing list](https://lists.spdx.org/g/spdx-tech) and ask questions about the document there.
```
Comment (nishakm) [inline docs/releases/v0_4_0.md]: You don't have to modify the command here in the releases. The `-l` command will still work here.

**Comment by nishakm** [comment]
@rnjudge This will be a breaking change. If we merge this now, our next release will be 2.0.0 not 1.1.0. Is this OK with you?

File: docs/releases/v0_4_0.md (hunk 1 above)
Comment (radmirnovii) [reply docs/releases/v0_4_0.md]: fixed

**Comment by radmirnovii** [comment]
> @rnjudge This will be a breaking change. If we merge this now, our next release will be 2.0.0 not 1.1.0. Is this OK with you?

yep

**Comment by rnjudge** [comment]
> @rnjudge This will be a breaking change. If we merge this now, our next release will be 2.0.0 not 1.1.0. Is this OK with you?

I am OK with this since we will be adding other command line options for dockerfile lock (granted, they are not breaking).

**Comment by rnjudge** [comment]
> We will also need a change in the Dockerfile ENTRYPOINT line:
> `ENTRYPOINT ["tern", "-q", "-b", "/hostmount"]

@radmirnovii wanted to make sure you saw change requested as well :) You'll need to include this in Tern's main `Dockerfile`.

**Comment by radmirnovii** [comment]
> > We will also need a change in the Dockerfile ENTRYPOINT line:
> > `ENTRYPOINT ["tern", "-q", "-b", "/hostmount"]
> 
> @radmirnovii wanted to make sure you saw change requested as well :) You'll need to include this in Tern's main `Dockerfile`.

fixed

**Comment by nishakm** [comment]
Looks good. It's a major change to the CLI (I will have to retrain my fingers :D). So let me make an announcement on the slack channel before merging.

**Pull request merged**